.env

```diff
@@ -1,3 +1,8 @@
 FACTOMD_HOST=localhost
 FACTOMD_PORT=8088
-ES_ADDRESS=<secret-entry-credit-address>
+FACTOMD_PROTOCOL=http
```

Comment:
Probably wise to call these MAINNET.

There are multiple networks possible. Supporting mainnet and testnet is enough for now though. But having a different naming scheme for mainnet could confuse people because they could also be interpreted as defaults for other networks.

Reply:
Good point. I refactored all the env variables to be `{NETWORK_NAME}_{VARIABLE_NAME}`. I think it's more consistent now.